depends/packages/libpng.mk

```diff
@@ -0,0 +1,23 @@
+package=libpng
+$(package)_version=1.6.34
+$(package)_download_path=http://ftp.osuosl.org/pub/libpng/src/libpng16/
```

Comment:
I'd rather use `https://downloads.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.xz` as the URL. Was this was a mirror you were redirected to, or were you trying to avoid SourceForge? I can't see this URL anywhere on http://www.libpng.org/pub/png/libpng.html.